src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -3,33 +3,55 @@ title: Dynamic Sampling
+  - /product/data-management-settings/server-side-sampling/current-limitations/
+  - /product/data-management-settings/server-side-sampling/sampling-configurations/
+  - /product/data-management-settings/dynamic-sampling/current-limitations/
+  - /product/data-management-settings/dynamic-sampling/sampling-configurations/
+description: "Learn about Sentry dynamically prioritizes your transaction data."
 ---
 
-<Include name="limited-avail-note.mdx" />
+Capturing a single [trace](/product/sentry-basics/tracing/distributed-tracing/) — which consists of one or more transactions — involves minimal overhead, but capturing traces for every page load or every API request might overload your system. Sentry automatically prioritizes your transactions, server-side, so that we can provide you with the most valuable insights about your application’s performance while also applying helpful prioritization to the data we retain for you.
 
-Capturing a single trace involves minimal overhead, but capturing traces for every page load or every API request may add undesirable load to your system. Sampling your transaction events allows you to better manage the number of transactions sent to Sentry so you can tailor your volume to your organization's needs.
+By default, we prioritize transaction data in the following ways:
 
-Sentry's **Dynamic Sampling** feature allows you to view more accurate performance metrics based on all the transaction events you send to Sentry (called _processed events_), while retaining a smaller number of more relevant transaction events that you can explore in full detail (called _indexed events_).
+<!-- prettier-ignore-start -->
 
-With dynamic sampling, you control which transactions Sentry retains by setting _server-side sampling_ rules and rates, so you see more of the transactions you want to explore further, and less of the ones you don’t. In addition, dynamic sampling allows you to manage your sampling configuration centrally, server-side, and lets you make changes without the need to reconfigure the SDK and redeploy your application.
+| Prioritize | Ignore |
+|------------|--------|
+| Latest release | Health checks |
+| Dev environments | |
 
-![The Dynamic Sampling page with a list of rules](dynamic-sampling.png)
+<!-- prettier-ignore-end -->
 
-> _[A trace](/product/sentry-basics/tracing/distributed-tracing/) represents the record of an entire operation you want to measure or track, like a page load, an instance of a user completing some action in your application, or a cron job in your backend. A trace consists of one or more transactions and the trace can be distributed across multiple services or projects._
+We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meanginful insights during your testing phase, we prioritize transaction data from these environments.
```

Comment:
```suggestion
We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
```